infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/configuration/impl/InfinispanRemoteConfiguration.java

```diff
@@ -173,14 +179,29 @@ public void initConfiguration(Map<?, ?> configurationMap, ServiceRegistryImpleme
 				.withDefault( InfinispanRemoteProperties.DEFAULT_SCHEMA_PACKAGE_NAME )
 				.getValue();
 
+		this.schemaFileName = propertyReader
+				.property( InfinispanRemoteProperties.SCHEMA_FILE_NAME, String.class )
+				.withDefault( InfinispanRemoteProperties.DEFAULT_SCHEMA_FILE_NAME )
```

Comment:
I will change it but, so that you know, you can apply a validation while reading the property.
Check the method `withValidator`